tools/talis/add.go

```diff
@@ -24,10 +25,21 @@ func addCmd() *cobra.Command {
 				return fmt.Errorf("failed to load config %q: %w", rootDir, err)
 			}
 
+			if provider == "" {
+				provider = "digitalocean"
+			}
+
 			switch nodeType {
 			case "validator":
 				for i := 0; i < count; i++ {
-					cfg = cfg.WithDigitalOceanValidator(region)
+					switch provider {
+					case "digitalocean":
+						cfg = cfg.WithDigitalOceanValidator(region)
+					case "googlecloud":
+						cfg = cfg.WithGoogleCloudValidator(region)
+					default:
+						return fmt.Errorf("unknown provider %q (supported: digitalocean, googlecloud)", provider)
```

Comment:
[FLUP] do we need a FLUP to add AWS or is that no longer necessary?

Ref: https://github.com/celestiaorg/celestia-app/issues/6093#issuecomment-3461596125

Reply:
AWS said that they don't like our workloads. Google cloud is enough and will cover our upcoming testing with very high throuput and strong machines. So we should be good